packages/intrepid2/assembly-examples/GRADGRADStandardAssembly.hpp

```diff
@@ -0,0 +1,147 @@
+//
+//  GRADGRADStandardAssembly.hpp
+//  Trilinos
+//
+//  Created by Roberts, Nathan V on 7/2/21.
+//
+
+#ifndef Intrepid2_GRADGRADStandardAssembly_hpp
+#define Intrepid2_GRADGRADStandardAssembly_hpp
+
+#include "JacobianFlopEstimate.hpp"
+
+//! version that uses the classic, generic Intrepid2 paths;
```

Comment:
Maybe here, and also in the other assembly files, you can add a sentence describing what you are assembling